tests/acceptance/10_files/can_set_sticky_bits_without_root_owning_directory.cf

```diff
@@ -27,7 +27,7 @@ bundle agent test
                    own.";
 
       "test_skip_unsupported"
-        string => "!windows";
+        string => "!has_stat";
```

Comment:
I split this in two, and made `windows` and `!has_stat` two different cases: `unsupported` and `needs_work`, respectively. They are subtly different and signal exactly what you mentioned in your last comment: On Windows this is not supported at all, whereas if stat is missing it is untested because the test is not good enough.